src/main/webapp/index.html

```diff
@@ -1,20 +1,57 @@
 <!DOCTYPE html>
 <html>
-  <head>
+
+<head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>GuideTech</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+
+    <link href="img/temp-icon.png" rel="icon">
```

Comment:
Do you have an image for this? Or do you want to remove this line?

Reply:
I had an image on the prototype that served as a logo for the page. It's not needed here, at least not yet, so I'll remove it! Thanks!